core/jvm/src/main/scala/zio/internal/ZScheduler.scala

```diff
@@ -380,17 +370,17 @@ private final class ZScheduler(autoBlocking: Boolean) extends Executor {
                   if ((worker ne self) && !worker.blocking) {
                     val size = worker.localQueue.size()
                     if (size > 0) {
-                      val runnables = worker.localQueue.pollUpTo(size - size / 2)
-                      if (runnables.nonEmpty) {
-                        runnable = runnables.head
-                        if (runnables.tail.nonEmpty) {
-                          localQueue.offerAll(runnables.tail)
-                        }
+                      val chunk     = worker.localQueue.pollUpTo(size - size / 2)
```

Comment:
🟢 Esthetic: I was preferring the previous name: `runnables`. It tells what this variable contains, not what it technically is (a chunk)